app/extras/queries/visible_motions.rb

```diff
@@ -7,7 +7,7 @@ def initialize(user: nil, groups: nil, group_ids: nil)
       group_ids = Array(groups).map(&:id)
     end
 
-    @relation = Motion.joins(:discussion => :group).merge(Group.published).preload(:discussion)
+    @relation = Motion.joins(discussion: :group).where('groups.archived_at IS NOT NULL').preload(:discussion)
```

Comment:
Does this seem okay to you, @robguthrie?

We were getting errors around `default_group_cover was not found on Motion` because we were merged a group relation and a motion relation.